examples/a10_timezone/a10_timezone.yml

```diff
@@ -0,0 +1,28 @@
+- name: Set timezone settings example playbook
+  connection: local
+  hosts: localhost
+  tasks:
+  - name: a10_timezone settings 
+    a10_timezone:
+      a10_host: "{{ a10_host }}"
+      a10_username: "{{ a10_username }}"
+      a10_password: "{{ a10_password }}"
+      a10_port: "{{ a10_port }}"
+      a10_protocol: "{{ a10_protocol }}"
+      timezone_index_cfg: 
+        timezone-index : "Asia/Calcutta"
+        nodst : true
+
+
+        # Supported Values for Timezone      
```

Comment:
This commend should be removed. They should refer to the AXAPI documentation for supported values.

Reply:
Resolved